lib/extensions.js

```diff
@@ -379,6 +403,53 @@ module.exports = function(realmConstructor) {
                     return classPermissions[0];
                 }
             },
+
+            subscriptions(name) {
+                let all_subscriptions = this.objects('__ResultSets');
+                if (typeof(name) !== 'string') {
```

Comment:
The docs mention that `name` is optional, but here it seems like we'll throw if it's not provided. Let's only check the type if it's not `undefined`.

Reply:
I believe OS will auto-generate a name if one isn't provided. It is in the form of `[Type] Query`

Reply:
Yeah, `name` is optional. Otherwise it will be hard to get *all* subscriptions.